job_templates/cyclic_cc_pt/config_fed_client.conf

```diff
@@ -0,0 +1,109 @@
+  app_config = ""
+
+  # Path to defined PyTorch network
+  # This assumes that there will be a "net.py" file with class name "Net", please modify accordingly
+  model_class_path = "net.Net"
+
+  # Client Computing Executors.
+  executors = [
+    {
+      # tasks the executors are defined to handle
+      tasks = ["train"]
+
+      # This particular executor
+      executor {
+
+        # Executor name : PTClientAPILauncherExecutor
+        # This is an executor for pytorch + Client API. The underline data exchange is using FilePipe.
+        path = "nvflare.app_opt.pt.client_api_launcher_executor.PTClientAPILauncherExecutor"
+
+        args {
+
+          # This executor take an component named "launcher"
+          launcher_id = "launcher"
+
+          # Timeout in seconds for waiting for a heartbeat from the training script. Defaults to 30 seconds.
+          # Please refer to the class docstring for all available arguments
+          heartbeat_timeout = 60
+
+          # data_exchange_path: is the directory location of the parameters exchange.
+          # If empty string, it will be set to the app_dir of the running job.
+          # You can also set it to an absolute path in your system.
+          data_exchange_path = ""
+
+          # format of the exchange parameters
+          params_exchange_format =  "pytorch"
+
+          # if the transfer_type is FULL, then it will be sent directly
+          # if the transfer_type is DIFF, then we will calculate the
+          # difference VS received parameters and send the difference
+          params_transfer_type = "DIFF"
+
+          # if train_with_evaluation is true, the executor will expect
+          # the custom code need to send back both the trained parameters and the evaluation metric
+          # otherwise only trained parameters are expected
+          train_with_evaluation = true
+
+          # if launch_once is true, the executor will only call launcher.launch_task once
+          # for the whole job, if launch_once is false, the executor will call launcher.launch_task
+          # everytime it receives a task from server
+          launch_once = true
+        }
+      }
+    }
+    {
+      # All tasks prefixed with cyclic_ are routed to CyclicClientController
+      tasks = ["cyclic_*"]
+      executor {
+        # client-side controller for cyclic workflow
+        path = "nvflare.app_common.ccwf.CyclicClientController"
+        args {
+          # maps to the train task of the PTClientAPILauncherExecutor
+          learn_task_name = "train"
+          persistor_id = "persistor"
+          shareable_generator_id = "shareable_generator"
+        }
+      }
+    }
+  ]
+
+  # Array of task data filters. If provided, it will control the data from client controller to client executor
+  # Filter direction (in, out, inout) can be set as since clients send tasks to each other, a task has both a sending (out) and a receiving (in) direction
```

Comment:
Thanks for the updates, we might need some documentation updates on this "filter" and direction and how they mean in client controlled workflows VS original server controlled workflows.

@nvkevlu @yhwen